akka-http-core/src/main/scala/akka/http/impl/engine/http2/BufferedOutletSupport.scala

```diff
@@ -38,9 +38,10 @@ private[http2] class BufferedOutlet[T](outlet: GenericOutlet[T]) extends OutHand
   outlet.setHandler(this)
 
   def onPull(): Unit = tryFlush()
-  def push(elem: T): Unit =
+  def push(elem: T): Unit = {
     if (outlet.canBePushed && buffer.isEmpty) doPush(elem)
     else buffer.addLast(elem)
+  }
```

Comment:
I think curlies are generally avoided in Akka HTTP sources when there is just a single expression